content/en/apps/guides/data/analytics/couch2pg-to-cht-sync-migration.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Migrating from Couch2pg to CHT Sync"
+weight: 1
+linkTitle: "Couch2pg to CHT Sync Migration"
+description: >
+  High-level instructions on how to migrate from Couch2pg to CHT Sync for data synchronization and analytics and what to consider during the migration process.
+relatedContent: >
+  core/overview/cht-sync
+  core/overview/data-flows-for-analytics/
+---
+
+This page outlines the process of migrating from [Couch2pg](https://github.com/medic/cht-couch2pg), to the new CHT Sync pipeline. One of the main changes in the new flow involves separating the syncing process from the data transformation, with DBT now handling the latter. This migration will require converting existing SQL views and tables into DBT models. One thing to note is that the schema for CHT Sync differs from Couch2pg, so the SQL views and tables will not be directly replaced by DBT models. For instructions on how to get started with DBT models refer to the [DBT Models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}}))
+
+## Key Considerations
+- **Server resources**: To minimize downtime, it is recommended to run both Couch2pg and CHT Sync in parallel during the migration process. With this in mind, ensure that the server and database resources are sufficient to handle the load.
+- **DBT Modelling**: Avoid the temptation to model new DBT models after existing SQL views and tables. Instead, take the opportunity to re-evaluate the data needs and design new models that are more efficient and effective. Think of what data needs to be shown and how it should be shown and use that to guide the design of the new models.
+- **Testing**: After migrating, thoroughly test the new DBT models to ensure that they are working as expected.
+- **Feedback**: Provide any feedback and create issues for errors or bugs encountered in the [CHT Sync](https://github.com/medic/cht-sync) and [CHT Pipeline](https://github.com/medic/cht-pipeline/) repos to make it easier for the next migration.
+
+## Migration Steps
+1. **Plan Migration**: Determine the scope of the migration, including the data sources, the data models, and the data transformations. Identify the SQL views and tables that need to be converted into DBT models.
```

Comment:
```suggestion
1. **Plan the migration**: Determine the scope of the migration, including the data sources, the data models, and the data transformations. Identify the existing SQL views, tables, and dashboards and assess what data you want to visualize. 
```

> Identify the SQL views and tables that need to be converted into DBT models.

This slightly conflicts with the abovementioned consideration about avoiding the temptation to convert existing views to DBT models directly. I am not 100% confident about my suggestion, but it should be about "think about what data you want to visualize and start from there"

Reply:
Your phrasing works better so going with it.